docs/pt/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,246 @@
+# Parâmetros do caminho da Url
+
+Você pode declarar os "parâmetros" ou "variáveis" com a mesma sintaxe utilizada pelo formato de strings do Python:
+
+```Python hl_lines="6-7"
+{!../../../docs_src/path_params/tutorial001.py!}
+```
+
+O valor do parâmetro que foi passado ao path `item_id` será passado para a sua função como o argumento `item_id`.
+
+Então, se você rodar este exemplo e for até <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a>, você verá a seguinte resposta:
+
+```JSON
+{"item_id":"foo"}
+```
+
+## Parâmetros de caminho com tipos
+
+Você pode declarar o tipo de um parâmetro na função usando as anotações padrões do Python:
+
+```Python hl_lines="7"
+{!../../../docs_src/path_params/tutorial002.py!}
+```
+
+Nesse caso, `item_id` está sendo declarado como um `int`.
+
+!!! Check
```

Comment:
I'm not sure what word are we using in Portuguese, but probably is not "Check". Maybe some page has it?
```suggestion
!!! Check "Something"
```

Reply:
O "check" tava certo. O que eu quis dizer é que da pra mudar a mensagem que aparece. Se você colocar `!!! check Algo`, a palavra "Algo" aparece na caixinha. Só que eu não lembro se a gente já usou essa palavra...

Reply:
Perfeito, entendi e fiz a alteração de acordo.